src/main/java/me/kermx/desirepaths/DesirePaths.java

```diff
@@ -78,97 +95,91 @@ private void onPlayerMove(PlayerMoveEvent event) {
+                            ridingHorseChance, ridingBoatChance, ridingPigChance, sprintingBlockBelowChance,
+                            sprintingBlockAtFeetChance, blockAtFeetSwitcherConfig, blockBelowSwitcherConfig);
                 }
             }
         }, 0L, attemptFrequency);
 
-        getServer().getPluginManager().registerEvents(this,this);
+        getServer().getPluginManager().registerEvents(this, this);
+
+        Bukkit.getConsoleSender()
+                .sendMessage(ChatColor.GOLD + ">>" + ChatColor.GREEN + " DesirePaths enabled successfully");
+
         // check if towny & worldguard are installed
         townyEnabled = Bukkit.getPluginManager().isPluginEnabled("Towny");
+        if (townyEnabled) {
+            Bukkit.getConsoleSender()
+                    .sendMessage(ChatColor.GOLD + ">>" + ChatColor.GREEN + " DesirePaths-Towny integration successful");
+        }
+
         worldGuardEnabled = Bukkit.getPluginManager().isPluginEnabled("WorldGuard");
-        if (townyEnabled && worldGuardEnabled) {
-            Bukkit.getConsoleSender().sendMessage(ChatColor.GOLD + ">>" + ChatColor.GREEN + " DesirePaths enabled successfully");
-            Bukkit.getConsoleSender().sendMessage(ChatColor.GOLD + ">>" + ChatColor.GREEN + " DesirePaths-Towny integration successful");
-            Bukkit.getConsoleSender().sendMessage(ChatColor.GOLD + ">>" + ChatColor.GREEN + " DesirePaths-WorldGuard integration successful");
-        } else if (townyEnabled){
-            Bukkit.getConsoleSender().sendMessage(ChatColor.GOLD + ">>" + ChatColor.GREEN + " DesirePaths enabled successfully");
-            Bukkit.getConsoleSender().sendMessage(ChatColor.GOLD + ">>" + ChatColor.GREEN + " DesirePaths-Towny integration successful");
-        } else if (worldGuardEnabled){
-            Bukkit.getConsoleSender().sendMessage(ChatColor.GOLD + ">>" + ChatColor.GREEN + " DesirePaths enabled successfully");
-            Bukkit.getConsoleSender().sendMessage(ChatColor.GOLD + ">>" + ChatColor.GREEN + " DesirePaths-WorldGuard integration successful");
-        } else {
-            Bukkit.getConsoleSender().sendMessage(ChatColor.GOLD + ">>" + ChatColor.GREEN + " DesirePaths enabled successfully");
+        if (worldGuardEnabled) {
+            Bukkit.getConsoleSender().sendMessage(
+                    ChatColor.GOLD + ">>" + ChatColor.GREEN + " DesirePaths-WorldGuard integration successful");
         }
     }
 
-
-    private void playerHandler(Player player, int noBootsChance, int leatherBootsChance, int hasBootsChance, int featherFallingChance, int ridingHorseChance, int ridingBoatChance, int ridingPigChance, int sprintingBlockBelowChance, int sprintingBlockAtFeetChance, List<String> blockAtFeetSwitcherConfig, List<String> blockBelowSwitcherConfig) {
-        if (player.getGameMode() != GameMode.SURVIVAL || player.hasPermission("desirepaths.exempt"))
```

Comment:
Deprecating `desirepaths.exempt` in favor of allow admin to toggle off users from console.